SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -2573,6 +2603,19 @@ void USpatialReceiver::MoveMappedObjectToUnmapped(const FUnrealObjectRef& Ref)
 	}
 }
 
+void USpatialReceiver::ClearPendingOwnerOnlyComponentsForEntity(Worker_EntityId EntityId)
+{
+	TArray<PendingAddComponentWrapper> QueuedComponents = ExtractAddComponents(&PendingOwnerOnlyComponents, EntityId);
+
+	const FDateTime Now = FDateTime::UtcNow();
```

Comment:
Don't do this in shipping ?

Reply:
Solved offline. Should be optimized out in a build without logging enabled.